blogs/2022/05/18/dev-container-cli.md

```diff
@@ -0,0 +1,131 @@
+
+```bash
+[88 ms] dev-containers-cli 0.1.0.
+[165 ms] Start: Run: docker build -f /home/node/vscode-remote-try-rust/.devcontainer/Dockerfile -t vsc-vscode-remote-try-rust-89420ad7399ba74f55921e49cc3ecfd2 --build-arg VARIANT=bullseye /home/node/vscode-remote-try-rust/.devcontainer
+[+] Building 0.5s (5/5) FINISHED
+ => [internal] load build definition from Dockerfile                       0.0s
+ => => transferring dockerfile: 38B                                        0.0s
+ => [internal] load .dockerignore                                          0.0s
+ => => transferring context: 2B                                            0.0s
+ => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/r  0.4s
+ => CACHED [1/1] FROM mcr.microsoft.com/vscode/devcontainers/rust:1-bulls  0.0s
+ => exporting to image                                                     0.0s
+ => => exporting layers                                                    0.0s
+ => => writing image sha256:39873ccb81e6fb613975e11e37438eee1d49c963a436d  0.0s
+ => => naming to docker.io/library/vsc-vscode-remote-try-rust-89420ad7399  0.0s
+[1640 ms] Start: Run: docker run --sig-proxy=false -a STDOUT -a STDERR --mount type=bind,source=/home/node/vscode-remote-try-rust,target=/workspaces/vscode-remote-try-rust -l devcontainer.local_folder=/home/node/vscode-remote-try-rust --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --entrypoint /bin/sh vsc-vscode-remote-try-rust-89420ad7399ba74f55921e49cc3ecfd2-uid -c echo Container started
+Container started
+{"outcome":"success","containerId":"f0a055ff056c1c1bb99cc09930efbf3a0437c54d9b4644695aa23c1d57b4bd11","remoteUser":"vscode","remoteWorkspaceFolder":"/workspaces/vscode-remote-try-rust"}
+```
+
+You can then run commands in this dev container:
+
+```bash
+devcontainer exec --workspace-folder <path-to-vscode-remote-try-rust> cargo run
+```
+
+This will compile and run the Rust sample, outputting:
+
+```bash
+[33 ms] dev-containers-cli 0.1.0.
+   Compiling hello_remote_world v0.1.0 (/workspaces/vscode-remote-try-rust)
+    Finished dev [unoptimized + debuginfo] target(s) in 1.06s
+     Running `target/debug/hello_remote_world`
+Hello, VS Code Remote - Containers!
+{"outcome":"success"}
+```
+
+Congrats, you’ve just run the dev container CLI and seen it in action!
+
+These steps are also provided in the CLI repo’s [readme](https://github.com/devcontainers/cli/blob/main/README.md).
+
+## How can I get involved?
+
+Your feedback is incredibly helpful in shaping and expanding the spec, and we’d love your feedback on the CLI so far. Please check out the CLI and let us know what you think. We’ve [opened an issue](https://github.com/devcontainers/cli/issues/7) specifically for folks to leave comments and questions, and you can also certainly also open new issues or PRs in the repo, just like other open-source projects.
```

Comment:
Would we also want to mention that people can provide feedback on the spec with a link to that repo's issues?

Reply:
That's a good idea. Added:

> We welcome your [feedback on the spec](https://github.com/devcontainers/spec/issues) anytime, and we’ll continue providing updates and info about what you can try out and when.